packages/polymath-ui/src/components/inputs/NumberInput/index.js

```diff
@@ -84,7 +84,7 @@ export class NumberInput extends Component<Props, State> {
 
   static getDerivedStateFromProps(props: Props, state: State) {
     const { oldValue } = state;
-    const { value, useBigNumbers, min, max, name } = props;
+    let { value, useBigNumbers, min, max, name } = props;
```

Comment:
Why was this changed to `let`?

Reply:
This was a mistake. Reverted with latest commit